### PR TITLE
Support externally extracted gDNA plates in production

### PR DESCRIPTION
**Pull request by AmandaBirmingham**
TL;DR: Fix for #369 Support "externally extracted" gDNA plates in production-like databases

During real-data testing against labman on qiita-rc, I discovered that the externally extracted gDNA plates functionality did not work because it depend on a number of dummy records in the database, and those records are added in the populate_test_db.sql script--which is not run when a production-like database is created.  This PR moves that functionality out of  populate_test_db.sql 
 and into db_patch_manual.sql--and then updates what-feels-like-zillions of hard-coded process and composition ids in the tests that are affected by having those records created before the test records rather than after them.

### CODE REVIEW

**Comment by ElDeveloper** [comment]
Thanks for such a thorough effort @AmandaBirmingham!
